cnxepub/collation.py

```diff
@@ -33,7 +33,8 @@ def easybake(ruleset, in_html, out_html):
     with respective read and write ability.
 
     """
-    html = etree.parse(in_html)
+    html_parser = etree.HTMLParser(encoding="utf-8")
```

Comment:
Yeppers, totally agree on the gnarliness of replacing XPath with CSS selectors (I tried doing that the 1st time I noticed XPaths in `cnx-easybake`.

Fortunately, it seems cssselect2 _does_ support namespace prefixes (at least parsing them): https://github.com/Connexions/cssselect2/blob/master/cssselect2/parser.py#L211